scylla-cql/src/frame/value_tests.rs

```diff
@@ -410,7 +416,7 @@ fn slice_batch_values() {
 
     {
         let mut request: Vec<u8> = Vec::new();
-        batch_values.write_nth_to_request(2, &mut request).unwrap();
+        it.write_next_to_request(&mut request).unwrap().unwrap();
         assert_eq!(request, vec![0, 1, 0, 0, 0, 1, 6]);
     }
 }
```

Comment:
I don't understand what you mean

Reply:
After running out of items the iterator should return `None`.
It would be nice to test that it does, and doesn't for example panic.

Something like:
```rust
assert_eq!(it.write_next_to_request(&mut request), None);
```

Reply:
I was hoping it would be applied to other tests as well, but alright let's just merge this.